### PR TITLE
Fix entry width and accessibility

### PR DESCRIPTION
**Pull request by jancborchardt**
- [x] Add max-width to recommendations entries to prevent only a single recommendation showing
- [x] Add hover and focus feedback 
- [x] Needs some JS to actually open the sidebar of the focused file on pressing Enter @ChristophWurst :)

Please check @nextcloud/designers 

Before:
Recommendations desktop befoer

After:
Recommendations desktop after
Recommendations mobile after

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,17 +1,11860 @@
...
-/*!
- * Vue.js v2.6.9
- * (c) 2014-2019 Evan You
- * Released under the MIT License.
+/******/ (function(modules) { // webpackBootstrap
```
Comment (gary-kim) [inline js/main.js]: This seems to be the dev mode version of the file.
Running `npm run build` will build the production version of the file.

**Comment by skjnldsv** [comment]
@juliushaertl we could use a webpack check ;)

File: js/main.js (hunk 1 above)
Comment (jancborchardt) [reply js/main.js]: Right – I'm always confused as to which commands to use and what to commit then. I'm on vacation now, so anyone is welcome to take over and add commits/amend, as also the "pressing Enter to open" has to be added.

Comment (gary-kim) [reply js/main.js]: Okay. I'll generate the webpack files and see if I can add the enter to open functionality.

File: src/components/RecommendedFile.vue [hunk 2]
```diff
@@ -21,7 +21,9 @@
 
 <template>
 	<a class="recommendation"
-	   @click.prevent="navigate">
+	   @click.prevent="navigate"
+	   v-on:keyup.enter.prevent="navigate"
```
Comment (skjnldsv) [inline src/components/RecommendedFile.vue]: Use shortand `@`

Comment (gary-kim) [reply src/components/RecommendedFile.vue]: Done.
Thank you for the review.

**Pull request merged**